### ISSUE TITLE
Add MathML elements to whitelist

### ISSUE THREAD

**Issue by alexmilowski** [gjtorikian/html-pipeline#336]
In looking at [lib/html/pipeline/sanitization_filter.rb](https://github.com/jch/html-pipeline/blob/master/lib/html/pipeline/sanitization_filter.rb), I see that all the MathML elements are missing from the whitelist. Presentational MathML is certainly innocuous markup. It could be added to this list without having harmful side effects.

Meanwhile, [MathML is part of HTML5](https://html.spec.whatwg.org/#mathml) and it is essential for creating accessible web pages with embedded Mathematics.

I'd be happy to provide a list of elements to add to the whitelist.

**Comment by gjtorikian** [comment]
Sounds good to me.

**Comment by alexmilowski** [comment]
I should just write a PR for this.

**Comment by gjtorikian** [comment]
A new (beta) release of HTML-Proofer has been released, v3.0.0.pre1. I tried to go back and address all the issues in this repo. For this specific issue, I've opted to keep the sanitization process the same. However, downstream consumers can easily make any change they want to the sanitization process; see [the updated README for more information](https://github.com/gjtorikian/html-pipeline#sanitization).
